### PR TITLE
Added Random Testimonial Generator

### LINKED ISSUE #2497: Random Testimonial Generator
**Describe the project you want to add with tech stack**
A random testimonial generator using HTML CSS and Javascript

**Screenshots (optional)**

image

### PR DESCRIPTION
**Pull request by PoulavBhowmick03**
# Title and Issue number 
Random Testimonial Generator #2497

Close #2497 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [] I have created a helpful and easy to understand `README.md`

https://github.com/pranjay-poddar/Dev-Geeks/assets/133862694/9b0812fe-37d6-4416-92c0-56094b95531b

### CODE REVIEW

**Comment by PoulavBhowmick03** [comment]
@MohitGupta121 please merge this

File: .vscode/settings.json
Comment (MohitGupta121) [inline .vscode/settings.json:1]: Again you changed this please revert these changes. Thanks!

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

**Comment by PoulavBhowmick03** [comment]
Done @MohitGupta121

**Comment by PoulavBhowmick03** [comment]
@MohitGupta121 fixed this, please merge it

**Comment by PoulavBhowmick03** [comment]
@pranjay-poddar @Saurabh-kumar09 @MohitGupta121

**Comment by PoulavBhowmick03** [comment]
@MohitGupta121 please merge this

**Review by MohitGupta121 — APPROVED** [review]
